src/chart/axisSide.js

```diff
@@ -0,0 +1,15 @@
+class AxisSide {
+    static LEFT = new AxisSide("LEFT")
+    static RIGHT = new AxisSide("RIGHT")
+
+    constructor(side) {
+        side = side.toLowerCase()
+        if (side !== "left" && side !== "right") {
+            throw new Error("side != left or right")
```

Comment:
Says "side not eq left OR right", but the if above states "not equal left AND not equal right".